README.md

```diff
@@ -79,6 +68,21 @@ Switch to this environment and use `poetry` to restore the package dependencies.
   $ .\env\Scripts\activate
   ```
 
+#### conda:
+
+
+<span style="color:orange">**⚠ WARNING: Due to licencing of Anaconda, it is recommended to use an alternative like [Miniforge](https://github.com/conda-forge/miniforge). This still uses the package manager conda and therefore the same commands can still be used. The same goes for [Miniconda](https://docs.anaconda.com/miniconda/) but you need to change the defaults channel to conda-forge!**
+</span>
```

Comment:
And how do you change the channel? It is probably blatantly obvious to seasoned pythoneers, but it does not leap to mind for this greyhair.